specification/ingest/delete_geoip_database/DeleteGeoipDatabaseRequest.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { RequestBase } from '@_types/Base'
+import { Ids } from '@_types/common'
+import { Duration } from '@_types/Time'
+
+/**
+ * Deletes a geoip database configuration.
+ * @rest_spec_name ingest.delete_geoip_database
+ * @availability stack since=8.15.0 stability=stable
+ * @availability serverless visibility=private
+ */
+export interface Request extends RequestBase {
+  path_parts: {
+    /**
+     * A comma-separated list of geoip database configurations to delete
+     */
+    id?: Ids
```

Comment:
@pquentin Is this really an optional parameter? This triggered a validation rule in my code generator.

The rule requires a mandatory parameter to be used in all available urls for an endpoint and an optional parameter to **not** be used in all available urls for an endpoint.

Reply:
We currently have only one url for this endpoint:

```json
"urls": [
        {
          "methods": [
            "DELETE"
          ],
          "path": "/_ingest/geoip/database/{id}"
        }
]
```

In other words: If that endpoint does not support an additional url like `"/_ingest/geoip/database"` (without the `{id}`), we should make the path parameter mandatory.